docker-compose/data-provider.config.yml

```diff
@@ -2,22 +2,22 @@ server:
   port: 8082
 orchestrator:
   pipelines:
-#    - name: gipod
-#      input:
-#        name: be.vlaanderen.informatievlaanderen.ldes.ldi.client.LdioLdesClient
-#        config:
-#          url: https://private-api.gipod.vlaanderen.be/api/v1/ldes/mobility-hindrances?generatedAtTime=2024-10-05T10:28:37.923Z
-#          sourceFormat: application/ld+json
-#      outputs:
-#        - name: be.vlaanderen.informatievlaanderen.ldes.ldi.RepositoryMaterialiser
-#          config:
-#            sparql-host: http://rdf4j-server:8080/rdf4j-server
-#            repository-id: test
-#            named-graph: http://gipod
-#        - name: be.vlaanderen.informatievlaanderen.ldes.ldio.LdioHttpOut
-#          config:
-#            content-type: application/n-quads
-#            endpoint: http://host.docker.internal:8084/api/gipod/members
+    - name: gipod
+      input:
+        name: be.vlaanderen.informatievlaanderen.ldes.ldi.client.LdioLdesClient
+        config:
+          url: https://private-api.gipod.vlaanderen.be/api/v1/ldes/mobility-hindrances?generatedAtTime=2024-10-05T10:28:37.923Z
+          sourceFormat: application/ld+json
+      outputs:
+        - name: be.vlaanderen.informatievlaanderen.ldes.ldi.RepositoryMaterialiser
+          config:
+            sparql-host: http://rdf4j-server:8080/rdf4j-server
+            repository-id: test
+            named-graph: http://gipod
+        - name: be.vlaanderen.informatievlaanderen.ldes.ldio.LdioHttpOut
+          config:
+            content-type: application/n-quads
+            endpoint: http://host.docker.internal:8084/api/gipod/members
 #    - name: verkeersmetingen-pipeline
 #      description: "Simple http in, version creation, http out pipeline allowing to pause output."
 #      input:
```

Comment:
If comment, do we need it?

Reply:
Was during testing that I wanted to exclude this stream, I will re-enable it again